# Russian translation for ubuntuone-client
# Copyright (c) 2009 Rosetta Contributors and Canonical Ltd 2009
# This file is distributed under the same license as the ubuntuone-client package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2009.
#
msgid ""
msgstr ""
"Project-Id-Version: ubuntuone-client\n"
"Report-Msgid-Bugs-To: FULL NAME <EMAIL@ADDRESS>\n"
"POT-Creation-Date: 2012-09-10 21:26+0000\n"
"PO-Revision-Date: 2012-03-30 18:24+0000\n"
"Last-Translator: Илья Калитко <Unknown>\n"
"Language-Team: Russian <ru@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=3; plural=n%10==1 && n%100!=11 ? 0 : n%10>=2 && "
"n%10<=4 && (n%100<10 || n%100>=20) ? 1 : 2;\n"
"X-Launchpad-Export-Date: 2012-09-17 18:32+0000\n"
"X-Generator: Launchpad (build 15966)\n"

#: ../ubuntuone/status/aggregator.py:51
msgid "Magicicada"
msgstr "Magicicada"

#: ../ubuntuone/status/aggregator.py:52
msgid "New cloud folder(s) available"
msgstr "Доступна новая облачная папка(ки)"

#: ../ubuntuone/status/aggregator.py:53
msgid "File synchronization completed."
msgstr "Синхронизация файлов завершена."

#: ../ubuntuone/status/aggregator.py:55
#, python-format
msgid "%(percentage_completed)d%% completed."
msgstr "%(percentage_completed)d%% завершено."

#: ../ubuntuone/status/aggregator.py:56
msgid "File synchronization in progress"
msgstr "Выполняется синхронизация файлов"

#: ../ubuntuone/status/aggregator.py:59
#, python-format
msgid ""
"There is no available space on the folder:\n"
"\"%s\" shared by %s"
msgstr ""
"Нет свободного места в папке:\n"
"«%s» открыл общий доступ: %s"

#: ../ubuntuone/status/aggregator.py:73
#, python-format
msgid "'%(filename)s' is being uploaded to your personal cloud."
msgstr "«%(filename)s» загружается в ваше личное облако."

#: ../ubuntuone/status/aggregator.py:79
#, python-format
msgid ""
"'%(filename)s' and %(other_files)d other file are being uploaded to your "
"personal cloud."
msgid_plural ""
"'%(filename)s' and %(other_files)d other files are being uploaded to your "
"personal cloud."
msgstr[0] ""
"«%(filename)s» и %(other_files)d другой файл загружаются в ваше личное "
"облако."
msgstr[1] ""
"«%(filename)s» и %(other_files)d других файла загружаются в ваше личное "
"облако."
msgstr[2] ""
"«%(filename)s» и %(other_files)d других файлов загружаются в ваше личное "
"облако."

#: ../ubuntuone/status/aggregator.py:90
#, python-format
msgid "'%(filename)s' is being downloaded to your computer."
msgstr "«%(filename)s» загружается на ваш компьютер."

#: ../ubuntuone/status/aggregator.py:96
#, python-format
msgid ""
"'%(filename)s' and %(other_files)d other file are being downloaded to your "
"computer."
msgid_plural ""
"'%(filename)s' and %(other_files)d other files are being downloaded to your "
"computer."
msgstr[0] ""
"«%(filename)s» и %(other_files)d другой файл загружаются на ваш компьютер."
msgstr[1] ""
"«%(filename)s» и %(other_files)d других файла загружаются на ваш компьютер."
msgstr[2] ""
"«%(filename)s» и %(other_files)d других файлов загружаются на ваш компьютер."

#: ../ubuntuone/status/aggregator.py:107
#, python-format
msgid "'%(filename)s' was uploaded to your personal cloud."
msgstr "«%(filename)s» загружен в ваше личное облако."

#: ../ubuntuone/status/aggregator.py:113
#, python-format
msgid ""
"'%(filename)s' and %(other_files)d other file were uploaded to your personal "
"cloud."
msgid_plural ""
"'%(filename)s' and %(other_files)d other files were uploaded to your "
"personal cloud."
msgstr[0] ""
"«%(filename)s» и %(other_files)d другой файл были загружены в ваше личное "
"облако."
msgstr[1] ""
"«%(filename)s» и %(other_files)d других файла были загружены в ваше личное "
"облако."
msgstr[2] ""
"«%(filename)s» и %(other_files)d других файлов были загружены в ваше личное "
"облако."

#: ../ubuntuone/status/aggregator.py:124
#, python-format
msgid "'%(filename)s' was downloaded to your computer."
msgstr "«%(filename)s» был загружен на ваш компьютер."

#: ../ubuntuone/status/aggregator.py:130
#, python-format
msgid ""
"'%(filename)s' and %(other_files)d other file were downloaded to your "
"computer."
msgid_plural ""
"'%(filename)s' and %(other_files)d other files were downloaded to your "
"computer."
msgstr[0] ""
"«%(filename)s» и %(other_files)d другой файл были загружены на ваш компьютер."
msgstr[1] ""
"«%(filename)s» и %(other_files)d других файла были загружены на ваш "
"компьютер."
msgstr[2] ""
"«%(filename)s» и %(other_files)d других файлов были загружены на ваш "
"компьютер."

#: ../ubuntuone/status/aggregator.py:191
#, python-format
msgid "A file was just made public at %(new_public_url)s"
msgstr "Файл был опубликован по адресу %(new_public_url)s"

#: ../ubuntuone/status/aggregator.py:204
#, python-format
msgid "%(event_count)d file was just made public."
msgid_plural "%(event_count)d files were just made public."
msgstr[0] "%(event_count)d файл опубликован."
msgstr[1] "%(event_count)d файлов опубликованы."
msgstr[2] "%(event_count)d файлов опубликовано."

#: ../ubuntuone/status/aggregator.py:212
msgid "A file is no longer published"
msgstr "Открытый доступ к файлу закрыт"

#: ../ubuntuone/status/aggregator.py:220
#, python-format
msgid "%(event_count)d file is no longer published."
msgid_plural "%(event_count)d files are no longer published."
msgstr[0] "Открытый доступ к %(event_count)d файлу закрыт."
msgstr[1] "Открытый доступ к %(event_count)d файлам закрыт."
msgstr[2] "Открытый доступ к %(event_count)d файлам закрыт."

#: ../ubuntuone/status/aggregator.py:235
#, python-format
msgid "Found %(event_count)d new cloud folder."
msgid_plural "Found %(event_count)d new cloud folders."
msgstr[0] "Найдена %(event_count)d облачная папка."
msgstr[1] "Найдено %(event_count)d облачные папки."
msgstr[2] "Найдено %(event_count)d облачных папок."

#: ../ubuntuone/status/aggregator.py:243
#, python-format
msgid ""
"New cloud folder available: '%(folder_name)s' shared by %(other_user_name)s"
msgstr ""
"Доступна новая папка в облаке: «%(folder_name)s» опубликована "
"%(other_user_name)s"

#: ../ubuntuone/status/aggregator.py:259
#, python-format
msgid "New cloud folder available: '%(folder_name)s'"
msgstr "Новая папка в облаке доступна: «%(folder_name)s»"

#: ../ubuntuone/status/aggregator.py:281
msgid "The connection to the server was lost."
msgstr "Соединение с сервером потеряно."

#: ../ubuntuone/status/aggregator.py:287
msgid "The connection to the server was restored."
msgstr "Соединение с сервером восстановлено."

#: ../data/emblem-ubuntuone-downloading.icon.in.h:1
msgid "Downloading"
msgstr "Загрузка"

#: ../data/emblem-ubuntuone-unsynchronized.icon.in.h:1
msgid "Unsynchronized"
msgstr "Не синхронизировано"

#: ../data/emblem-ubuntuone-uploading.icon.in.h:1
msgid "Uploading"
msgstr "Отправка данных"
